tree/ntuple/src/RNTupleDescriptorFmt.cxx

```diff
@@ -124,7 +124,14 @@ void ROOT::RNTupleDescriptor::PrintInfo(std::ostream &output) const
          }
          const auto &pageRange = cluster.second.GetPageRange(column.second.GetPhysicalId());
          auto idx = cluster2Idx[cluster.first];
+         std::uint64_t prevLocatorOffset = 0;
+         std::uint64_t locatorOffset;
          for (const auto &page : pageRange.GetPageInfos()) {
+            locatorOffset = page.GetLocator().GetType() == ROOT::RNTupleLocator::ELocatorType::kTypeDAOS
+                               ? page.GetLocator().GetPosition<RNTupleLocatorObject64>().GetLocation()
+                               : page.GetLocator().GetPosition<std::uint64_t>();
+            if (locatorOffset == prevLocatorOffset)
+               continue;
             nBytesOnStorage += page.GetLocator().GetNBytesOnStorage();
             nBytesInMemory += page.GetNElements() * elementSize;
```

Comment:
Aliased pages should probably be counted towards `nBytesInMemory` because (at least for the moment) we actually load and decompress them multiple times